### PR TITLE
Fix displaying of explain after paging and improve looks.

### PR DESCRIPTION
**Pull request by EreMaijala**
- Aligns the bar with the icon.
- Moves the bar after any hierarchy link.
- Adds a bit of top margin for breathing room.
- Ensures that the bar fits the page on narrower screens.
- Sandal: re-enables single-column display of result-links in xs width.

### CODE REVIEW

**Comment by EreMaijala** [comment]
Note about the sandal-specific change: apparently there used to be a narrow right-hand side column for result links that displayed only icons, but it was already broken for cart links and relevance, and there was no way to make it work for the latter.

File: module/VuFind/src/VuFind/AjaxHandler/GetSearchResults.php
Comment (demiankatz) [inline module/VuFind/src/VuFind/AjaxHandler/GetSearchResults.php:222]: I wonder if it's worth aliasing the Params class as ParamsHelper or QueryParams to make this less potentially confusing -- I had to read the code carefully to understand this change (which does make perfect sense upon examination). Maybe better done as a follow-up action, if it's worth doing it all, since it's not really related to the rest of the work here. Just an observation!

**Review by demiankatz — COMMENTED** [review]
This looks fine to me -- just one minor suggestion below.

I've requested a review from @ThoWagen in case he has any specific concerns, since he's a key stakeholder for the Explain functionality.

File: module/VuFind/src/VuFind/AjaxHandler/GetSearchResults.php
Comment (demiankatz) [reply module/VuFind/src/VuFind/AjaxHandler/GetSearchResults.php:222]: I see you've renamed the parameter instead of the type, which does the job just as well -- thanks!

Comment (EreMaijala) [reply module/VuFind/src/VuFind/AjaxHandler/GetSearchResults.php:222]: I decided to rename both the Params and the actual parameter variable for clarity. Ended up with ParamsHelper since QueryParams could be confused to params in the query string.

Comment (demiankatz) [reply module/VuFind/src/VuFind/AjaxHandler/GetSearchResults.php:222]: ...and now you've done the type as well. Excellent all around! :-)

**Comment by demiankatz** [comment]
Thanks, @EreMaijala, this looks good to me but I'll hold off on approving until @ThoWagen has a chance to weigh in.

**Comment by ThoWagen** [comment]
The changes of this PR look good to me. I just noticed some problems in combination with the "Saved in" box.

Bildschirmfoto vom 2024-03-11 16-30-28
It is hard to keep the controls of different records apart.

Also if the screen width is between 768px and 990px the name of the list sticks out of the box.
Bildschirmfoto vom 2024-03-11 16-31-01

I you don't want to fix this in this PR I could look into it.

**Comment by EreMaijala** [comment]
@ThoWagen Good call! I tweaked it a bit, but if you'd like to do any further refinements, please feel free to open an additional PR.

**Comment by demiankatz** [comment]
Thanks, @EreMaijala and @ThoWagen -- I'll wait to see what @ThoWagen wants to do before merging this.

**Review by ThoWagen — APPROVED** [review]
Great, I think this is ready to be merged.

**Review by demiankatz — APPROVED** [review]
Definitely a step in the right direction -- thanks, @EreMaijala and @ThoWagen!

**Pull request merged**
